app/views/overview/_service-instance-row.html

```diff
@@ -6,8 +6,8 @@
     <div class="list-pf-content">
       <div class="list-pf-name">
         <h3>
-          <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
-          <div ng-bind-html="row.apiObject.metadata.name | highlightKeywords : row.state.filterKeywords" class="list-row-longname"></div>
```

Comment:
#1471 was merged, so I rebased and squashed. No files changed by `grunt build` anymore.